### PR TITLE
Continuous integration

### PR DESCRIPTION
**Pull request by Timple**
Ready to be reviewed and merged, please Squash and merge

Todo: Use three scripts instead of travis file.
- [x] Create docker image
- [x] Install package X branch Y (can later be called from other repos)
- [x] Compile package X

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,5 +15,5 @@ jobs:
       - docker login -u="$DOCKER_HUB_USERNAME" -p="$DOCKER_HUB_PASSWORD"
       # push the new Docker image to the Docker registry only after acceptance of pull request
       - 'if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then
-            docker push $IMAGE_NAME
+            docker push $IMAGE_NAME;
```
Comment (LoyVanBeek) [inline .travis.yml]: Oh, duh :-)

**Comment by Timple** [comment]
Copied long term goals to issue https://github.com/tue-robotics/tue-env/issues/95

Ready to be reviewed (again)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+USER amigo
+WORKDIR /home/amigo
+
+# Remove interactive check from bashrc
+RUN sed -e s/return//g -i ~/.bashrc
+
+# Setup tue env
+RUN mkdir -p ~/.tue
+ADD ./ /home/amigo/.tue/
```
Comment (LoyVanBeek) [inline Dockerfile]: What does this do? AFAIK ```ADD``` adds something from the computer running the Dockerfile, right? Is this then reproducible?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+RUN sudo chown -R amigo:amigo /home/amigo/.tue/
+ENV CI=true
+ENV LANG=C.UTF-8
+RUN  /home/amigo/.tue/installer/scripts/bootstrap-ros-kinetic
```
Comment (LoyVanBeek) [inline Dockerfile]: Ah, right, the ```installer``` subdir is added and then run. Documenting that would help

**Pull request merged**